internal/watch.go

```diff
@@ -152,9 +162,17 @@ func (opts buildAndWatch) Run() error {
 						close(abort)
 						return err
 					}
+				case <-ctx.Done():
+					close(abort)
```

Comment:
reviewer note: this looks right b/c of the following return, so you can't unintentionally close the abort channel twice 👍🏻